pxtsim/runtime.ts

```diff
@@ -37,6 +37,15 @@ namespace pxsim {
             return Date.now();
         }
 
+        export function nowUs(): number {
+            return (performance as any).now() ||
```

Comment:
When I try to use process.hrtime() it says  " Cannot find name 'process' ", What should I do?